### ISSUE TITLE
Sticky Header

### ISSUE THREAD

**Issue by geektutor** [DSC-Unilag/ShortLink#14]
Can we have a version of the header where it is responsive and sticky on scroll - https://dscunilag.dev/ShortLink/

**Comment by Emmanuerl** [comment]
yes we can, its actually already reusable i stated the instructions in the
index.html file comment just before the nav tag. as for the sticky feature,
i'll fix it asap

**Comment by geektutor** [comment]
Fixed?

**Comment by Emmanuerl** [comment]
yes, just made a pr
